packages/core-modules/contracts/storage/ElectionStorage.sol

```diff
@@ -68,7 +68,11 @@ contract ElectionStorage {
         /**
          * @dev Voting data for the current election
          */
-        ElectionData electionData;
+        mapping(uint => ElectionData) electionData;
+        /**
+         * @dev Index pointing to the latest electionDta
+         */
+        uint electionDataIndex;
```

Comment:
Could we express this notion of "latest" in this property's name?
I.e. `latestElectionIdx` or `currentElectionIdx`.

Reply:
Fixed